src/kits/arm/arm_node.cpp

```diff
@@ -298,12 +329,28 @@ class ArmNode {
     // (We use the last position command for smoother motion)
     Eigen::VectorXd last_position = arm_.lastFeedback().getPositionCommand();
 
+    if(use_ik_seed_) {
+      last_position = ik_seed_;
+    }
+
     // For each waypoint, find the joint angles to move to it, starting from the last
     // waypoint, and save into the position vector.
     if (end_tip_directions) {
       // If we are given tip directions, add these too...
+      //Eigen::VectorXd elbow_up(6); elbow_up << 0, 3.14 / 4.0, 3.14 / 2.0, 3.14 / 4.0, -3.14, 3.14 / 2.0;
       for (size_t i = 0; i < num_waypoints; ++i) {
         last_position = arm_.solveIK5Dof(last_position, xyz_positions.col(i), end_tip_directions->col(i));
+        ROS_INFO_STREAM("IK Solution: " << last_position[0] << " | "
+                                        << last_position[1] << " | "
+                                        << last_position[2] << " | "
+                                        << last_position[3] << " | "
+                                        << last_position[4] << " | "
+                                        << last_position[5]);
+        auto fk_check = arm_.FK3Dof(last_position);
+        ROS_INFO_STREAM("FK computed pose:" << fk_check[0] << ", " << fk_check[1] << ", " << fk_check[2]);
+        ROS_INFO_STREAM("target pose:" << xyz_positions.col(i)[0] << ", " << xyz_positions.col(i)[1] << ", " << xyz_positions.col(i)[2]);
+        //auto mag_diff = (last_position - fk_check).norm();
+        //ROS_INFO_STREAM("Distance between commanded and computed end effector pose: " << mag_diff);
```

Comment:
I like having some diagnostics here -- perhaps format a nice ROS_WARN for a significant difference here, and maybe only show IK angles then?  I'm not sure what the recommendation is in terms of how much diagnostic output should be from a node.